custom_components/tuya_local/devices/windmill_airconditioner.yaml

```diff
@@ -0,0 +1,55 @@
+name: Windmill Air Conditioner
```

Comment:
Top level name should be without branding, as often other devices will also match the config.